### ISSUE TITLE
UNCAUGHT EXCEPTION FOR VIEW: tx_mkforms_forms_Base FormId:tx_seminars_pi1_registration_editor

### ISSUE THREAD

**Issue by peerlight** [oliverklee/ext-seminars#521]
Message: : Property $TSFE->loginUser is not in use anymore as this information is now stored within the frontend.user aspect. in /var/www/clients/client1/web6/web/typo3_src-9.5.14/typo3/sysext/frontend/Classes/Controller/TypoScriptFrontendController.php on line 5307

I get this message after last registration form submit step, FE shows 
```
Oops, an error occurred! 
<h2>MKFORMS:</h2><p><strong><b></b>: TypoScript property config.titleTagFunction is deprecated - Please use the new TitleTag API to create custom title tags. Deprecated in version 9, will be removed in version 10 in  
```

with
- TYPO3 9.5.14
- PHP 7.2.x
- seminars 3.0.0 - 3.0.1
- onetimeaccount 4.0.1 - 4.1.0
- rn_base 1.11.1 - 1.11.2
- mkforms 9.5.2

stacktrace
```
<li>TYPO3\CMS\Frontend\Http\RequestHandler->handle#46</li> 
<li>TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController->INTincScript#187</li> 
<li>TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController->generatePageTitle#3775</li> 
<li>TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController->logDeprecatedTyposcript#3695</li>
 <li>trigger_error#4325</li> 
 <li>tx_ameosformidable->__catchEvalException#</li> 
 <li>tx_ameosformidable->mayday#3436</li> 
 <li>tx_mkforms_util_Div::mayday#3299</li> 
 <li>tx_rnbase_util_Debug::getDebugTrail#236</li>
```
in seminars sendParametersToThankYouAfterRegistrationPageUrl = 0/1 makes no difference

**Comment by oliverklee** [comment]
Hi @peerlight, thanks for the bug report!

To rule other things out, could you please check whether `config.titleTagFunction` is set in your TypoScript setup (it shouldn't be)? Thanks!

**Comment by peerlight** [comment]
Hi, tried this before (as cs_seo still uses config.titleTagFunction)

results in 
```
Oops, an error occurred!
<h2>MKFORMS:</h2><p><strong><b></b>: TemplateService->getFileName() will be removed in TYPO3 v10.0. Use FilePathSanitizer->sanitize() of EXT:frontend instead. in <b>/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/TypoScript/TemplateService.php</b> on line <b>1439</b><br /><hr /><span style='font-family: Verdana; font-size: 9px;color: green;'>!NULL!</span><hr/><code><span style="color: #000000"> </span> </code></strong></p><span class='notice'><strong>XML: </strong> </span><br /><span class='notice'><strong>MKFORMS Version: </strong>v9.5.2</span><br /><span class='notice'><strong>Total exec. time: </strong>0.421 sec</span><br /><br /><span class="notice"><strong>debug trail: </strong></span><ol> <li>{closure}#25</li> <li>TYPO3\CMS\Core\Http\AbstractApplication->run#24</li> <li>TYPO3\CMS\Frontend\Http\Application->handle#120</li> <li>TYPO3\CMS\Core\Http\AbstractApplication->handle#69</li> <li>TYPO3\CMS\Core\Http\MiddlewareDispatcher->handle#108</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#67</li> <li>TYPO3\CMS\Frontend\Middleware\TimeTrackerInitialization->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#49</li> <li>TYPO3\CMS\Core\Middleware\NormalizedParamsAttribute->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#58</li> <li>TYPO3\CMS\Frontend\Middleware\PreprocessRequestHook->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#57</li> <li>TYPO3\CMS\Frontend\Middleware\EidHandler->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#50</li> <li>TYPO3\CMS\Frontend\Middleware\TypoScriptFrontendInitialization->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#90</li> <li>TYPO3\CMS\Frontend\Middleware\FrontendUserAuthenticator->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#79</li> <li>TYPO3\CMS\Frontend\Middleware\BackendUserAuthenticator->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#95</li> <li>TYPO3\CMS\Frontend\Middleware\SiteResolver->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#68</li> <li>TYPO3\CMS\Frontend\Middleware\SiteBaseRedirectResolver->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#86</li> <li>TYPO3\CMS\Frontend\Middleware\StaticRouteResolver->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#66</li> <li>TYPO3\CMS\Redirects\Http\Middleware\RedirectHandler->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#70</li> <li>TYPO3\CMS\Frontend\Middleware\PageResolver->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#171</li> <li>TYPO3\CMS\Frontend\Middleware\MaintenanceMode->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#53</li> <li>JWeiland\Maps2\Middleware\InitFeSessionMiddleware->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#84</li> <li>TYPO3\CMS\Frontend\Middleware\PageArgumentValidator->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#99</li> <li>TYPO3\CMS\Frontend\Middleware\PrepareTypoScriptFrontendRendering->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#118</li> <li>TYPO3\CMS\Frontend\Middleware\ShortcutAndMountPointRedirect->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#65</li> <li>TYPO3\CMS\Frontend\Middleware\ContentLengthResponseHeader->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#45</li> <li>In2code\Luxletter\Middleware\LuxletterLink->process#138</li> <li>class@anonymous/__MYPATH__/typo3_src-9.5.14/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php0x7f83967db056->handle#67</li> <li>TYPO3\CMS\Frontend\Middleware\OutputCompression->process#138</li> <li>TYPO3\CMS\Frontend\Http\RequestHandler->handle#46</li> <li>TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController->INTincScript#187</li> <li>TYPO3\CMS\Core\Page\PageRenderer->renderJavaScriptAndCssForProcessingOfUncachedContentObjects#3788</li> <li>TYPO3\CMS\Core\Page\PageRenderer->renderJavaScriptAndCss#1884</li> <li>TYPO3\CMS\Core\Page\PageRenderer->executePreRenderHook#1932</li> <li>TYPO3\CMS\Core\Utility\GeneralUtility::callUserFunction#2789</li> <li>call_user_func_array#3592</li> <li>WapplerSystems\WsScss\Hooks\RenderPreProcessorHook->renderPreProcessorProc#</li> <li>TYPO3\CMS\Core\TypoScript\TemplateService->getFileName#120</li> <li>trigger_error#1439</li> <li>tx_ameosformidable->__catchEvalException#</li> <li>tx_ameosformidable->mayday#3436</li> <li>tx_mkforms_util_Div::mayday#3299</li> <li>tx_rnbase_util_Debug::getDebugTrail#236</li></ol>
```
I cropped the last error message. This one is uncropped.

**Comment by oliverklee** [comment]
Could you please retry after temporarily deactivating the `cs_seo` extension?

**Comment by oliverklee** [comment]
Possible there are two issues here at play: `FrontEndController::loginUser` and `config.titleTagFunction`.

**Comment by peerlight** [comment]
deactivating cs_seo does not change anything as `config.titleTagFunction` is already cleared

switching this off in rn_base Extension Configuration helps for the moment:
```
basic.forceException4Mayday (boolean)
basic.dieOnMayday (boolean) 
```

**Comment by oliverklee** [comment]
Okay, I'll take care of #526, and after that, you can retest.

**Comment by oliverklee** [comment]
@peerlight I have released seminars 3.0.2 today that includes the fix #526. Could you please retest with this version? Thanks!

**Comment by oliverklee** [comment]
No feedback for more than 6 months. Closing.

@peerlight Please feel free to reopen this ticket with new information.
